### ISSUE TITLE
Websites down/offline

### ISSUE THREAD

**Issue by ademarsj** [Nozbe/WatermelonDB#1856]
The  websites are down:

https://watermelondb.dev/ (the docs)
https://watermelondb.vercel.app/ (the demo)

**Comment by radex** [comment]
@ademarsj the main website seems to work for me - can you check again?

**Comment by ademarsj** [comment]
You're right, it's a problem with my network provider, using VPN works fine.
Sorry for the issue.
